### PR TITLE
post: nodejs v18

### PR DESCRIPTION
**Pull request by navarroaxel**

### CODE REVIEW

File: posts/navarroaxel/01.nodejs-18.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
+---
+title: 'Node.js 18: fetch API, Test Runner module, and more'
+published: false
+description: Node.js v18 was released and we can found a few amazing stuff in this major release.
+tags: 'javascript, node, webdev, programming'
+cover_image: ./assets/nodejs-18-cover.png
+---
+
+This week Node.js v18 was released and we can found a few amazing stuff in this major release. Let's check the most important ones. 🚀
```
Comment (ilaborde) [inline posts/navarroaxel/01.nodejs-18.md:9]: we can **find**

Comment (navarroaxel) [inline posts/navarroaxel/01.nodejs-18.md:9]:
```suggestion
This week Node.js v18 was released and we can find new amazing features in this major release. Let's check the most important ones. 🚀
```

File: posts/navarroaxel/01.nodejs-18.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+Node.js v18 comes with V8 engine v10.1 which is part of Chromium v101, in spite of the versions included in Node.js v17 that didn't include these features:
+
+* The [`findLast` and `findLastIndex` array methods](https://v8.dev/features/finding-in-arrays).
+* Improvements to the [`Intl.Locale` API](https://v8.dev/blog/v8-release-99#intl.locale-extensions).
+* The [`Intl.supportedValuesOf` function](https://v8.dev/blog/v8-release-99#intl.locale-extensions).
+
+## Conclusion
+
+In this post we've only mentioned the most important changes included in this release. Remember that this version will be promoted to LTS in October of this year.
```
Comment (ilaborde) [inline posts/navarroaxel/01.nodejs-18.md:77]: maybe you can simplify this line to:

> We've only mentioned the most important changes included in this release. This version will be promoted to LTS in October of this year.

File: posts/navarroaxel/01.nodejs-18.md (hunk 1 above)
Comment (navarroaxel) [reply posts/navarroaxel/01.nodejs-18.md:9]:
```suggestion
This week Node.js v18 was released and we can find new amazing features in this major release. Let's check the most important ones. 🚀
```

File: posts/navarroaxel/01.nodejs-18.md (hunk 2 above)
Comment (navarroaxel) [reply posts/navarroaxel/01.nodejs-18.md:77]:
```suggestion
We've only mentioned the most notable changes in this release. Node.js v18 will be promoted to LTS in October of this year.
```

**Review by pabloromeo — APPROVED** [review]

**Review by ilaborde — APPROVED** [review]

**Pull request merged**